bft/html/builder.py

```diff
@@ -186,7 +186,10 @@ def create_function_info(
     supplements: SupplementsFile,
     dialects: DialectsLibrary,
 ) -> FunctionInfo:
-    name = "_".join(func.name.split('_')[1:])
+    if func.name.startswith('aggregate'):
```

Comment:
It looks like we're doing this over and over.  Perhaps create a function to find the simplified name?